commands/show/cluster/command.go

```diff
@@ -608,3 +609,22 @@ func formatNodePoolDetails(nodePools *models.V5GetNodePoolsResponse) []string {
 
 	return rows
 }
+
+func formatClusterLabels(labels map[string]string) []string {
+	formattedClusterLabels := []string{color.YellowString("Labels:|") + "-"}
+
+	isFirstLine := true
+
+	for key, value := range labels {
+		if strings.Contains(key, "giantswarm.io") == false {
```

Comment:
Can you please extract the `giantswarm.io` token into a constant at the top of the file?